src/components/LandingPage/HeroSection.tsx

```diff
@@ -0,0 +1,198 @@
+
+const HERO_TILE_STYLES = "mx-10 mb-20 md:mb-0 lg:mt-10";
+const LINK_STYLES =
+  " cursor-pointer opacity-95 md:my-4 flex justify-start font-bold md:justify-center";
+
+const WEBPAGE_CONTAINER =
+  "hidden md:block lg:block rounded-t-md bg-[#00D3A9] opacity-95 ";
+const WINDOW_BUTTONS_SVG = "/svgs/heroSection/window_control_buttons.svg";
+const COUNTDOWN_CONTAINER = "flex justify-center my-12";
+
+const COUNTDOWN_BACKGROUND =
+  " bg-[#A689FF] text-white py-10 px-4 rounded-lg md:w-[8rem] md:h-[11rem] lg:w-[10rem] lg:h-[11rem] relative ";
+const NUMBER_STYLES =
+  "flex flex-col justify-center items-center font-bold text-[20px] md:text-[58px] lg:text-[60px]";
+const LABEL_STYLES =
+  " flex flex-col justify-center items-center uppercase text-1xl";
+const HORIZONTAL_LINE =
+  "absolute left-0 right-0 top-[50%] -translate-y-1/2 transform border-b-2 border-white opacity-95";
+
+interface HeroSectionProps {
+  eventName: string;
+  eventYear: string;
+  eventBlurb: string;
+  eventDate: number;
+}
+
+const HeroSectionDetails: HeroSectionProps = {
+  eventName: "Hack the Change",
+  eventYear: "2024",
+  eventBlurb:
+    "Hack the Change 2024 is a hybrid two-day for-charity hackathon with the mission of coding a better world together.",
+  eventDate: 1731394799, //UNIX Time stamp: Nov 10, 2024
+};
+
+const HeroSectionTile = (props: HeroSectionProps) => {
+  //Destructure props object to access specific properties
+  const { eventName, eventYear, eventBlurb, eventDate } = props;
+
+  const [hackathonTime, setHackathonTime] = useState(false);
+  const [days, setDays] = useState(0);
+  const [hours, setHours] = useState(0);
+  const [minutes, setMinutes] = useState(0);
+  const [seconds, setSeconds] = useState(0);
+
+  const formatDate = (unixTimestamp: number): string => {
+    const date = new Date(unixTimestamp * 1000);
+    return date.toLocaleDateString();
+  };
+
+  const eventDateFormatted = formatDate(eventDate);
+
+  useEffect(() => {
+    const target = new Date(eventDateFormatted);
+
+    const interval = setInterval(() => {
+      const now = new Date();
+      const difference = target.getTime() - now.getTime();
+
+      const d = Math.floor(difference / (1000 * 60 * 60 * 24));
+      setDays(d);
+
+      const h = Math.floor(
+        (difference % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60),
+      );
+      setHours(h);
+
+      const m = Math.floor((difference % (1000 * 60 * 60)) / (1000 * 60));
+      setMinutes(m);
+
+      const s = Math.floor((difference % (1000 * 60)) / 1000);
+      setSeconds(s);
+      if (d <= 0 && h <= 0 && m <= 0 && s <= 0) {
+        setHackathonTime(true);
+      }
+    }, 1000);
+    return () => clearInterval(interval);
+  }, []);
+
+  return (
+    <div className={HERO_TILE_STYLES}>
+      <div>
+        <h1
+          className="mt-16 flex-wrap text-5xl font-black text-[#FFFF] drop-shadow-lg md:text-center md:text-6xl"
+          style={{
+            textShadow: `
```

Comment:
could you put this in an object above? The styling makes it hard to read the other code since it's a big block, something like 

```
const HEADER_STYLE = {
	textShadow: `
		-2px -2px 0 #7055FD, 
		2px -2px 0 #7055FD, 
		-2px 2px 0 #7055FD, 
		2px 2px 0 #7055FD,
		-2px -2px 0 #7055FD,
		2px -2px 0 #7055FD,
		-2px 2px 0 #7055FD,
		2px 2px 0 #7055FD
	`
}
```